packages/package_kronik_2_02/tool_dependencies.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<tool_dependency>
+    <package name="kronik" version="2.02">
+        <install version="1.0">
+            <actions>
+		<action type="download_by_url" target_filename="kronik.zip">http://proteome.gs.washington.edu/software/hardklor/public/kronik.zip</action>
```

Comment:
Does not seem to be versioned, maybe also cargo-port? https://github.com/galaxyproject/cargo-port